tests/test_scripts.py

```diff
@@ -84,13 +88,42 @@ def test_proto_conversion(tmpdir):
     assert p.stdout.startswith(b'{\n  "meta": ') or p.stdout.startswith(b'{\r\n  "meta": ')
 
 
-def test_detect_duplicate_features():
-    new_rule_path = "collection/credit-card/parse-credit-card-information.yml"
-    args = [
-        get_rules_path(),
-        os.path.join(get_rules_path(), new_rule_path),
-    ]
-    expected_overlaps = 49
+def run_detect_duplicate_features(rule_path):
+    # rule_path = "collection/credit-card/parse-credit-card-information.yml"
+    args = [get_rules_path(), rule_path]
     script_path = get_script_path("detect_duplicate_features.py")
-    p = run_program(script_path, args)
-    assert p.returncode == expected_overlaps
+    args = [sys.executable] + [script_path] + args
+    print(f"running: '{args}'")
+    return subprocess.run(args)
+
+
+def test_detect_duplicate_features(z9324d_extractor, tmpdir):
+    # tests a single rule can be loaded successfully
+    RULE_CONTENT = textwrap.dedent(
```

Comment:
analogous to this I'd create a test rule set we can test the overlap on

Reply:
Created a List of Rules. Then created a temporary directory containing rules.yml files for the test rules and running the script on this temp dir and every rule in it.

Reply:
Nice, this looks good now.